README.md

```diff
@@ -66,6 +66,48 @@ start(app, config);
 
 Alternatively, you may create your own Express instance, add middleware beforehand and pass that instance as parameter when calling `universal.app(app)`.
 
+#### Koa middleware
+
+Instead of express the renderer can return koa compatible middleware. Unlike with express you have to create your own koa instance and handle static routing
+yourself.
+
+```javascript
+import koa from 'koa';
+import compress from 'koa-compress';
+import favicon from 'koa-favicon';
+import serve from 'koa-static';
+import path from 'path';
+
+/* Minimum config for koa example:
+ export default {
+  server: {
+    rendererWebFramework: 'koa'
+  }
+ }
+ */
+import userConfig from './../config/universal-redux.config.js';
+
+import { configure, renderer } from 'universal-redux';
+
+const config = configure(userConfig);
+const app = koa();
+
+app.use(compress());
+
+if (config.server.favicon) {
+  app.use(favicon(path.resolve(config.server.favicon)));
+}
+
+const maxAge = config.server.maxAge || 0;
+app.use(serve(path.resolve(config.server.staticPath), {maxage: maxAge}));
+
+app.use(renderer(config));
+
+app.listen(config.server.port);
+console.info('==> 💻  Open http://%s:%s in a browser to view the app.', config.server.host, config.server.port);
```

Comment:
@bartolkaruza Rather than have this code example in the README for universal-redux, could you provide a working koa starter and link to the server logic in that project? I'd also like to verify by cloning and running that myself.